taca_ngi_pipeline/deliver/deliver_grus.py

```diff
@@ -249,9 +300,26 @@ def deliver_project(self):
                 logger.exception(e)
                 status = False
                 return status
+            if self.include_project:
+                for prj, pobj in all_projects.iteritems():
+                    if pobj.pi_email != self.pi_email:
+                        logger.error("PI ({}) for project '{}' is not same as PI ({}) for main project '{}'."\
+                                     " So aborting delivery.".format(pobj.pi_email, prj, self.pi_email, self.projectid))
+                        status = False
+                        return status
         else:
-            logger.warning("email for PI for project {} specified by user: {}".format(self.projectid,
+            logger.warning("Email of PI for project {} specified by user: {}".format(self.projectid,
                         self.pi_email))
+            if self.include_project:
+                question = "\nUser specified PI email '{}' will be used for all included projects '{}' as well. "\
+                           "Is it OK ? ".format(self.pi_email, ", ".join(self.include_project))
+                if proceed_or_not(question):
```

Comment:
Will these questions all be asked up front or between sample deliveries (i.e. will there be waiting times)? Perhaps it would be good to be able to force things through.

Reply:
All the questions will be asked before the actual hard staging begins and `force` was not added in order to have avoid human error as much as possible. Because for `GRUS` delivery, the delivery project are created based upon PI email. So when one includes multiple project in single delivery they should  be certain that all project belong to same PI (so we dont deliver data to whom it doesn't belong 😅 )

This is checked automatically when `--pi-email` is not provided in command line, but provided this question is prompted to remind the user to makes sure all project belong to same PI